plugins/base/src/test/kotlin/transformerBuilders/PageTransformerBuilderTest.kt

```diff
@@ -197,7 +197,7 @@ class PageTransformerBuilderTest : BaseAbstractTest() {
                 assertContains(generatedFiles.keys, "scripts/symbol-parameters-wrapper_deferred.js")
 
                 val scripts = generatedFiles.getValue("root/test/-test/-test.html").let { Jsoup.parse(it) }.select("script")
-                val deferredScriptSources = scripts.filter { it.hasAttr("defer") }.map { it.attr("src") }
+                val deferredScriptSources = scripts.filter { element -> element.hasAttr("defer") }.map { it.attr("src") }
```

Comment:
See https://github.com/Kotlin/dokka/issues/2754#issuecomment-1347703088 for the explanation